src/cpu/flexcpu/simple_dataflow_cpu.cc

```diff
@@ -249,6 +249,45 @@ SimpleDataflowCPU::requestInstruction(const RequestPtr& req,
     return true;
 }
 
+bool
+SimpleDataflowCPU::requestMemRead(const RequestPtr& req, ThreadContext* tc,
+                                  MemCallback callback_func)
+{
+    // TODO may need to enqueue request if port is busy
+    PacketPtr pkt = Packet::createRead(req);
+
+    uint8_t* mailbox = new uint8_t[req->getSize()]
+    pkt->dataStatic<uint8_t>(mailbox);
```

Comment:
What is this mailbox? It should at least have a comment, but I have a feeling there's a better way to accomplish what you're trying to do here.

Reply:
It's basically how TimingSimpleCPU does it. It allocates some memory with new in order to have something to pass to the Packet for dataDynamic/Static, then deletes it afterward. Tbh since the lifespan of a packet is well-defined, I actually think that it's the cleanest way to manage it. I guess this was why I was asking if there was a guaranteed recvTimingResp call for every sendTimingReq sent successfully.